packages/onboarding-ui/src/forms/NewAccountForm/NewAccountForm.tsx

```diff
@@ -0,0 +1,169 @@
+import {
+  EmailInput,
+  FieldGroup,
+  Field,
+  ButtonGroup,
+  Button,
+  PasswordInput,
+  TextInput,
+  Box,
+  CheckBox,
+} from '@rocket.chat/fuselage';
+import { ReactElement, useEffect } from 'react';
+import { useForm, SubmitHandler, Validate } from 'react-hook-form';
+import { Trans, useTranslation } from 'react-i18next';
+
+import Form from '../../common/Form';
+
+export type NewAccountPayload = {
+  name: string;
+  email: string;
+  password: string;
+  confirmPassword: string;
+  agreement?: boolean;
+};
+
+type NewAccountFormProps = {
+  initialValues?: Omit<NewAccountPayload, 'password'>;
+  validateEmail: Validate<string>;
+  validatePassword: Validate<string>;
+  validateConfirmationPassword: Validate<string>;
+  onSubmit: SubmitHandler<NewAccountPayload>;
+};
+
+const NewAccountForm = ({
+  validateEmail,
+  validatePassword,
+  validateConfirmationPassword,
+  onSubmit,
+}: NewAccountFormProps): ReactElement => {
+  const { t } = useTranslation();
+
+  const {
+    register,
+    handleSubmit,
+    formState: { isValidating, isSubmitting, isValid, errors },
+    setFocus,
+  } = useForm<NewAccountPayload>({ mode: 'onChange' });
+
+  useEffect(() => {
+    setFocus('name');
+  }, [setFocus]);
+
+  return (
+    <Form onSubmit={handleSubmit(onSubmit)}>
+      <Form.Container>
+        <FieldGroup>
+          <Field>
+            <Field.Label>
+              {t('form.newAccountForm.fields.name.label')}
+            </Field.Label>
+            <Field.Row>
+              <TextInput
```

Comment:
What do you think about use `useUniqueId` to pass id to the inputs activating them clicking on the label?